### DISCUSSION TITLE
Developer Meeting?

### DISCUSSION THREAD

**Discussion by ArneVoss** [su2code/SU2#2010] [General]
When is the developer meeting? I just joined the video call but I was the only participant... Did I miss something?
https://meet.jit.si/SU2_DevMeeting

**Comment by bigfooted** [comment]
Hi, 
On odd weeks we have the meeting on Tuesday, 19:00 CET and on even weeks it is Wednesday at 16:00 CET
If you have development questions you can also use the SU2 slack channel.

**Comment by pcarruscag** [comment]
You can join our slack via this link https://join.slack.com/t/su2devteam/shared_invite/zt-af0uuqf8-8XNExKMV9G~UVsnkvi5uVA

**Comment by ArneVoss** [comment]
Great, thank you!
